include/predict/predict.h.in

```diff
@@ -370,6 +370,17 @@ predict_julian_date_t predict_next_los(const predict_observer_t *observer, predi
  **/
 double predict_doppler_shift(const predict_observer_t *observer, const predict_orbit_t *orbit, double downlink_frequency);
 
+/**
+ * Calculate squint angle for satellite, i.e. the offset of the satellite antenna with respect to the QTH antenna. Is defined only for SDP4 satellites for now.
+ *
+ * \param observer Point of observation
+ * \param orbit Current state of satellite orbit
+ * \param alon Attitude longitude in radians (describes orientation of the satellite at apogee)
+ * \param alat Attidue latitude in radians (see above)
+ * \return Squint angle in radians. Will output nan if the satellite is not an SDP4 satellite
+ **/
```

Comment:
This looks very similar to the code removed from sdp4_predict(). A \copyright GPLv2+ tag would be appropriate.